packages/react-native-lockdown/src/index.js

```diff
@@ -0,0 +1,127 @@
+const { warn } = console
+const { assign } = Object
+
+const path = require('node:path')
+
+const defaultGetRunModuleStatement = (moduleId) => `__r(${moduleId})`
+
+const ENTRY_FILE_MODULE_ID = 0
+
+const warnAboutAnomalies = () => {
+  const deferredWarnings = {
+    calledOnceOnly:
+      'LavaMoat: getRunModuleStatement was only called once instead of at least twice in quick succession, with react-native InitializeCore first and bundle entry second. This is suspicious. Lockdown might not be in effect.',
+    neverCalledWithEntryModule:
+      'LavaMoat: getRunModuleStatement was not called with moduleId 0. Lockdown was not applied.',
+  }
+
+  // Emit warnings only after Metro finishes bundling
+  process.on('exit', () => {
+    Object.values(deferredWarnings).forEach((message) => {
+      if (message) warn(message)
+    })
+  })
+
+  return function inspectCallLog(callLog) {
+    if (callLog.length > 1) {
+      deferredWarnings['calledOnceOnly'] = false
```

Comment:
```suggestion
      deferredWarnings.calledOnceOnly = false
```

I am surprised `prettier` didn't do this for you.  do me a favor and execute `npm run setup` from the lavamoat workspace root 😄

Reply:
ran `npm run setup` and `Format Document (forced)` but still no autofix 🤔 updated anyway